### PR TITLE
docs: fix typos and formatting

### PR DESCRIPTION
**Pull request by peterj**
fix typos and formatting in the docs.

### CODE REVIEW

**Comment by qmonnet** [comment]
Could you please amend your commit to fix the description (you have the description of three different commits in it)? You may drop the `Co-authored-by:` tag here; if you prefer to keep it, I'd appreciate if you could use my work email (`quentin@isovalent.com`) rather than personal one that GitHub uses by default.

---

As far as I understand, GitHub is having internal issues that would explain most of the CI errors we observe.

The Netlify errors are not related to GitHub though:

```
6:16:37 PM: /opt/build/repo/Documentation/network/kubernetes/kubeproxy-free.rst:73: WARNING: Explicit markup ends without a blank line; unexpected unindent.
6:16:37 PM: /opt/build/repo/Documentation/network/kubernetes/kubeproxy-free.rst:94: WARNING: Literal block ends without a blank line; unexpected unindent.
```

Can you please look into this?

File: Documentation/network/kubernetes/kubeproxy-free.rst [hunk 1]
```diff
@@ -65,22 +65,37 @@ the token returned by ``kubeadm init``
...
 For existing installations with ``kube-proxy`` running as a DaemonSet, remove it
-by using the following commands below. **Careful:** Be aware that this will break
-existing service connections. It will also stop service related traffic until the
-Cilium replacement has been installed:
+by using the following commands below.
+
+.. warning::
+   Be aware that removing ``kube-proxy`` will break existing service connections. It will also stop service related traffic
+   until the Cilium replacement has been installed:
+by using the following commands below.
```
Comment (qmonnet) [inline Documentation/network/kubernetes/kubeproxy-free.rst]: There's one indent error here, causing one of the errors in Netlify.

File: Documentation/network/kubernetes/kubeproxy-free.rst [hunk 2]
```diff
@@ -65,22 +65,37 @@ the token returned by ``kubeadm init``
...
+
+.. warning::
+   Be aware that removing ``kube-proxy`` will break existing service connections. It will also stop service related traffic
+   until the Cilium replacement has been installed:
```
Comment (qmonnet) [inline Documentation/network/kubernetes/kubeproxy-free.rst]: Duplicated contents. Please double-check your commit.

File: Documentation/network/kubernetes/kubeproxy-free.rst [hunk 3]
```diff
@@ -65,22 +65,37 @@ the token returned by ``kubeadm init``
...
+Next, generate the required YAML files and deploy them. 
+.. important:: 
+
+   Make sure you correctly set your ``API_SERVER_IP`` and ``API_SERVER_PORT`` below with the control-plane node IP address
+   and the kube-apiserver port number reported by ``kubeadm init`` (Kubeadm will use port ``6443`` by default).
+Next, generate the required YAML files and deploy them. 
+.. important:: 
+
+   Make sure you correctly set your ``API_SERVER_IP`` and ``API_SERVER_PORT`` below with the control-plane node IP address
+   and the kube-apiserver port number reported by ``kubeadm init`` (Kubeadm will use port ``6443`` by default).
```
Comment (qmonnet) [inline Documentation/network/kubernetes/kubeproxy-free.rst]: Same here, indent error on line 94, plus duplicated contents. Please double-check.

**Review by qmonnet — CHANGES_REQUESTED** [review]

**Comment by peterj** [comment]
> Could you please amend your commit to fix the description (you have the description of three different commits in it)? You may drop the `Co-authored-by:` tag here; if you prefer to keep it, I'd appreciate if you could use my work email (`quentin@isovalent.com`) rather than personal one that GitHub uses by default.
> 
> As far as I understand, GitHub is having internal issues that would explain most of the CI errors we observe.
> 
> The Netlify errors are not related to GitHub though:
> 
> ```
> 6:16:37 PM: /opt/build/repo/Documentation/network/kubernetes/kubeproxy-free.rst:73: WARNING: Explicit markup ends without a blank line; unexpected unindent.
> 6:16:37 PM: /opt/build/repo/Documentation/network/kubernetes/kubeproxy-free.rst:94: WARNING: Literal block ends without a blank line; unexpected unindent.
> ```
> 
> Can you please look into this?

Looks like a merge gone wrong -- it should be fixed now. I think GH added the co-authored portion, modified it with the your work email.

File: Documentation/network/kubernetes/kubeproxy-free.rst [hunk 4]
```diff
@@ -65,22 +65,27 @@ the token returned by ``kubeadm init``
...
+   $ kubectl -n kube-system delete cm kube-proxy
+   $ # Run on each node with root permissions:
+   $ iptables-save | grep -v KUBE | iptables-restore
 
 .. include:: ../../installation/k8s-install-download-release.rst
 
-Next, generate the required YAML files and deploy them. **Important:** Make sure you correctly set your
-``API_SERVER_IP`` and ``API_SERVER_PORT`` below with the control-plane node IP address and the kube-apiserver port number reported by ``kubeadm init`` (Kubeadm will use port ``6443`` by default).
+Next, generate the required YAML files and deploy them. 
+.. important::
```
Comment (qmonnet) [inline Documentation/network/kubernetes/kubeproxy-free.rst]:
```suggestion
Next, generate the required YAML files and deploy them.

.. important::
```

You need a blank line here, or Sphinx considers your directive as part of the paragraph:

image

Please make sure that the formatting changes render correctly in [the preview](https://deploy-preview-25365--docs-cilium-io.netlify.app/network/kubernetes/kubeproxy-free.html).

The rest looks better now, thanks!

File: Documentation/network/kubernetes/kubeproxy-free.rst [hunk 5]
```diff
@@ -65,22 +65,27 @@ the token returned by ``kubeadm init``
...
+
+   Make sure you correctly set your ``API_SERVER_IP`` and ``API_SERVER_PORT`` below with the control-plane node IP address
+   and the kube-apiserver port number reported by ``kubeadm init`` (Kubeadm will use port ``6443`` by default).
```
Comment (qmonnet) [inline Documentation/network/kubernetes/kubeproxy-free.rst]: Nit: We might still want to better wrap these lines (or the ones from the warning above).

File: Documentation/network/kubernetes/kubeproxy-free.rst [hunk 6]
```diff
@@ -65,22 +65,29 @@ the token returned by ``kubeadm init``
...
 
 For existing installations with ``kube-proxy`` running as a DaemonSet, remove it
-by using the following commands below. **Careful:** Be aware that this will break
-existing service connections. It will also stop service related traffic until the
-Cilium replacement has been installed:
+by using the following commands below.
+
+.. warning::
+   Be aware that removing ``kube-proxy`` will break existing service connections. It will also stop service related traffic
+   until the Cilium replacement has been installed:
```
Comment (qmonnet) [inline Documentation/network/kubernetes/kubeproxy-free.rst]: One last nit: the colon no longer makes sense here.

```suggestion
   until the Cilium replacement has been installed.
```

Please also re-wrap the warning.

**Review by qmonnet — APPROVED** [review]
Thanks

**Comment by peterj** [comment]
@qmonnet what are the next steps for this PR? I see the branch if not up to date with main, but I am hesitant to click merge and get into the same cycle as I did with the previous PR (ie. unsigned commits, unclean commit history etc).  Is update with rebase the right option at this point?

**Comment by qmonnet** [comment]
Hi, no need to rebase against `main`, unless there were a conflict (in which case I'd recommend fixing and rebasing locally before updating the branch, rather than using GitHub's button).

Here we're just missing a review from cilium/sig-k8s. @youngnick could you please take a look?

**Review by youngnick — APPROVED** [review]
Sorry about the delay here.

**Comment by qmonnet** [comment]
Doc change only, and related tests passed. The full CI suite is not required, we're good to go

**Pull request merged**